packages/grant-explorer/src/features/api/passport.ts

```diff
@@ -4,6 +4,75 @@ import {
   PassportResponse,
   PassportState,
 } from "common";
+import { useEffect, useState } from "react";
 
 export { submitPassport, fetchPassport, PassportState };
 export type { PassportResponse };
+
+export function usePassport({ address }: { address: string }) {
+  const [passport, setPassport] = useState<PassportResponse | undefined>();
+  const [, setError] = useState<Response | undefined>();
+
+  const [passportState, setPassportState] = useState<PassportState>(
+    PassportState.LOADING
+  );
+
+  useEffect(() => {
+    setPassportState(PassportState.LOADING);
+
+    const PASSPORT_COMMUNITY_ID =
+      process.env.REACT_APP_PASSPORT_API_COMMUNITY_ID;
+    const PASSPORT_THRESHOLD = 0;
+
+    if (address && PASSPORT_COMMUNITY_ID) {
```

Comment:
This is not clearly defined from a product perspective rn. It will function, but the passport checking features will be unavailable, so a silent failure. Will be addressed by buidltime env var validation.